### ISSUE TITLE
How to use it? 

### ISSUE THREAD

**Issue by BiltuDas1** [yogeshdcool/prodigy#1]
Where are the usage docs?
Means what parameters should I pass to the program, or how to open programs respective to their mime type or file extensions or url?
Should I do like this?
> prodigy example.zip

It is not mention clearly into the README.md, it would be harder to understand for those whose are not familer with xdg-open.

**Comment by yogeshdcool** [comment]
>Should I do like this?
> 
> > prodigy example.zip  
yes it works like that I will update the readme soon and thanks for your feedback

**Comment by BiltuDas1** [comment]
And another suggestion, the configuration file looks weird, you can make it like it real configuration file on Linux, and use the source command to load the settings, just how we do bashrc/zshrc.
https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html
Example by me:
https://github.com/BiltuDas1/qsort/blob/bc3c85b73a958f3d14d05a309734d13fa97d1797/conf/qsort.conf

**Comment by yogeshdcool** [comment]
Thanks for the suggestion I will do something about it

**Comment by yogeshdcool** [comment]
> the configuration file looks weird  

Thanks for the suggestion I looked into your links and learnt a easy way to write configs I will do somethink soon...

**Comment by BiltuDas1** [comment]
> terminal="alacritty"
launcher="rofi -dmenu -l 1 -location 2 -p open-with"

Why you set it as default when there's no related configuration found? You should have set an exception handling instead of that. Cause I don't think every linux os have `alacriitty` or `rofi` installed, I also don't know what are these, so set an Exception handling instead of setting this configuration as default.

**Comment by BiltuDas1** [comment]
And can you make the code looks like a style of OOPS(since you can't do this into shell programming because shell doesn't support class and objects) , It would be easier to contribute, sometimes I didn't understood from where the code is executing and where it going.

**Comment by yogeshdcool** [comment]
> > terminal="alacritty"
> > launcher="rofi -dmenu -l 1 -location 2 -p open-with"
> 
> Why you set it as default when there's no related configuration found?  

added some info about that now
